test/metabase/api/setup_test.clj

```diff
@@ -243,13 +246,15 @@
                      (setup! assoc-in [:prefs :site_locale] "en-EN")))))
 
     (testing "user"
-      (testing "first name"
-        (is (= {:errors {:first_name "value must be a non-blank string."}}
-               (setup! m/dissoc-in [:user :first_name]))))
+      (with-redefs [api.setup/*allow-api-setup-after-first-user-is-created* true]
+        (binding [*expected-status* 200]
+          (testing "first name may be nil"
+            (is (= true (boolean (:id (setup! m/dissoc-in [:user :first_name])))))
+            (is (= true (boolean (:id (setup! assoc-in [:user :first_name] nil))))))
```

Comment:
Since names can be nil, I wanted to reflect that in the test. I could have alternatively just removed these few tests, but wasn't sure if that is a good idea.